sdk/python/jobs/automl-standalone-jobs/automl-distributed-classification-task-creditcard/classification-distributed-lightgbm.ipynb

```diff
@@ -0,0 +1,1472 @@
+    "|**trial_timeout_minutes**|Maximum time in minutes that each trial (child job) can run for before it terminates. If not specified, a value of 1 month or 43200 minutes is used.|\r\n",
+    "|**max_trials**|The maximum number of trials/runs each with a different combination of algorithm and hyperparameters to try during an AutoML job. If not specified, the default is 1000 trials. If you are setting the `enable_early_termination=True` the number of trials will be smaller.|\r\n",
+    "|**max_concurrent_trials**|Represents the maximum number of trials (children jobs) that would be executed in parallel. It's a good practice to set this number equal to the number of nodes in your cluster.|\r\n",
+    "|**enable_early_termination**|Whether to enable early termination if the score is not improving over 10 iterations. Early stopping window starts only after first 20 iterations. This means that the first iteration where stopping can occur is the 31st.|\r\n",
+    "|**max_nodes**|The maximum number of nodes to use for `distributed` training. For classification/regression tasks, the minimum value that needs to be set is 4.|\r\n",
+    "\r\n",
+    "\r\n",
+    "##### set_training() parameters:\r\n",
+    "This is an optional configuration method to configure training parameters.\r\n",
+    "\r\n",
+    "|Property|Description|\r\n",
+    "|-|-|\r\n",
+    "|**allowed_training_algorithms**|A list of Regression algorithms to try out as base model for model training in an experiment. If it is omitted or set to None, then all supported algorithms are used during experiment|\r\n",
+    "|**enable_model_explainability**|A flag to turn on model explainability like feature importance, of best model evaluated by Automated ML system.|\r\n",
+    "|**training_mode**|[Experimental] The training mode to use. The possible values are-<ul>`distributed`- enables distributed training for supported algorithms.<ul>`non_distributed`- disables distributed training.<ul>`auto`- Currently, it is same as non_distributed. In future, this might change.<br><i>*Note*: This parameter is in public preview and may change in future.</i>|\r\n",
```

Comment:
Please fix the markdown:

\- \`distributed\`- enables distributed training for supported algorithms.
\- \`non_distributed\`- disables distributed training.
\- \`auto\`- Currently, it is same as non_distributed. In future, this might change.

\<i\>\*Note\*: This parameter is in public preview and may change in future.\</i\>

Reply:
To render without extra spaces please consider using the following:
<ul><li>`distributed`- enables distributed training for supported algorithms.</li><li>`non_distributed`- disables distributed training.</li><li>`auto`- Currently, it is same as non_distributed</li></ul>

Reply:
Fixed it. Thank you.